cosmos/settings.py

```diff
@@ -60,6 +60,24 @@
     },
 }
 
+# Cache
+# https://docs.djangoproject.com/en/3.0/ref/settings/#caches
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+    }
+}
+if secret_settings.secrets["CACHE"]["USE_REDIS"]:
```

Comment:
I dont think we necessarily want to make it an option, as supporting two different implementation of the celery stuff especially is not worth it, so i think it is best if we just switch it over fully.

Reply:
This cache is not only used for celery, but also for caching ORM and page renders. This is already configured by default, changing to redis for this cache will only improve performance, and not produce any breakage, as it's a Django thing.

Reply:
Yeah i agree we should redis, im just saying we should not give an option whether or not to use redis, but we should just switch to redis, making it so that everybody has to use redis. This also goes for the celery broker url below. This because i dont want to have to support two different setups, so we should just always use redis.